src/less/pagination/ds6/pagination.less

```diff
@@ -100,13 +95,7 @@
 .pagination--fluid {
     .pagination__items {
         flex-grow: 1;
-        height: 3rem; // a11y: needs to be relative to the text zoom level
         max-width: 512px;
-        padding: 4px 0 4px 8px;
-
-        > li {
-            line-height: 3rem; // a11y: needs to be relative to the text zoom level
```

Comment:
We should ensure zoom still works.

Reply:
Testing now

Reply:
Yes. Zoom is still working with this change.